### PR TITLE
Osmosis token swaps example

### PR DESCRIPTION
**Pull request by kruspy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -61,4 +61,4 @@ typings/
 .env
 
 # next.js build output
-.next
+.next
```
Comment (pienkowb) [inline .gitignore:64]: No newline

File: package.json [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
   "dependencies": {
     "babel-polyfill": "^6.26.0",
-    "babel-register": "^6.26.0",
-    "truffle": "^5.0.4",
-    "truffle-contract": "^4.0.5",
-    "truffle-hdwallet-provider": "^1.0.4"
+    "babel-register": "^6.26.0"
   }
-}
+}
```
Comment (pienkowb) [inline package.json:23]: No newline

File: schema.graphql [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
+  denom: String
 }
+
+type TokenSwap @entity {
+  id: ID!
+  sender: String
+  poolId: String
+  tokenIn: Token
+  tokenOut: Token
+}
```
Comment (pienkowb) [inline schema.graphql:13]: No newline

**Comment by kruspy** [comment]
> I think we had some Typescript config in other examples. Do we want to add it here?

What config do you mean ?

**Comment by pienkowb** [comment]
> > I think we had some Typescript config in other examples. Do we want to add it here?
> 
> What config do you mean ?

This one: https://github.com/figment-networks/example-subgraph/blob/cosmos-block-filtering/tsconfig.json

File: README.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
-# Example Subgraph
+# Token Swaps Example
```
Comment (pienkowb) [inline README.md:1]: Osmosis Token Swaps Example

**Review by pienkowb — APPROVED** [review]

**Pull request merged**